lib/normalize_title_filter.js

```diff
@@ -74,58 +75,30 @@ module.exports = function(hyper, req, next, options, specInfo) {
                         return res;
                     });
                 } else {
-                    /* TODO: we don't want to enable redirects yet, so log the redirect instead
-                     and disable caching of the response
-
-                     rp.title = result;
-                     var pathBeforeTitle = specInfo.path
-                     .substring(0, specInfo.path.indexOf('{title}'));
-                     pathBeforeTitle = new URI(pathBeforeTitle, rp, true).toString();
-                     var pathAfterTitle = req.uri.toString().replace(pathBeforeTitle, '');
-                     var backCount = (pathAfterTitle.match(/\//g) || []).length;
-                     var backString = Array.apply(null, { length: backCount }).map(function() {
-                     return '../';
-                     }).join('');
-                     var pathPatternAfterTitle = specInfo.path
-                     .substring(specInfo.path.indexOf('{title}') - 1);
-                     var contentLocation = backString
-                     + new URI(pathPatternAfterTitle, rp, true).toString().substr(1);
-                     return P.resolve({
-                     status: 301,
-                     headers: {
-                     location: contentLocation
-                     }
-                     });*/
-                    if (rp.title.replace(/ /g, '_') !== resultText) {
-                        // Log all of these, should be relatively rare.
-                        hyper.log('warn/normalize/non_space', {
-                            msg: 'Normalized non-space chars',
-                            from: rp.title,
-                            to: resultText,
-                        });
-                    } else if (Math.random() < 0.05) {
-                        // 5% chance of logging to bound volume.
-                        hyper.log('warn/normalize/space', {
-                            msg: 'Normalized requested title',
-                            from: rp.title,
-                            to: resultText,
-                        });
-                    }
-                    nextRequest = next(hyper, req)
-                    .then(function(res) {
-                        if (res) {
-                            res.headers = res.headers || {};
-                            res.headers['cache-control'] = 'no-cache';
+                    rp.title = resultText;
```

Comment:
Minor: This assignment is now shared between the branches, so we could pull it out.